src/services/GenerateEML.ts

```diff
@@ -4,7 +4,9 @@ import { IRecipient, IUser } from '../types';
 export default (recipient: IRecipient, user: IUser) => {
   const messageId = new Date().getUTCMilliseconds();
   const userFullName = user.firstName + ' ' + user.lastName;
-  const recipientFullName = recipient.firstName + ' ' + recipient.lastName;
+  const firstName = recipient.firstName || '';
+  const lastName = recipient.lastName || '';
```

Comment:
const { firstName = '', lastName = '' }  = recepient;

Reply:
Updated :) Remember, the word is `recipient`, not `recepient` :)